bin/lib/book.ml

```diff
@@ -255,6 +255,22 @@ let make_simple_page file =
   ] in
   return (main_template ~title_bar:title_bar ~content ())
 
+let make_content_page ?(repo_root=".") () : string Deferred.t =
```

Comment:
maybe name this `make_tex_page` to be more explicit about what it does?

Reply:
Actually I am not sure to understand what `content.tex` is supposed to contain? Is that just the list of inputs? So maybe  name it `inputs.tex` instead?